packages/paste-core/components/inline-code/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {InlineCode} from '../src';
+
+describe('InlineCode', () => {
```

Comment:
suggestion: let's add a small test check that this component renders as a `code` html element, in case that ever tries to change in the future.

Reply:
sounds good!